### PR TITLE
Start bandit study on Beta and Nightly

### PR DESCRIPTION
**Pull request by LorenzoMinto**

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -544,6 +544,38 @@
                 "channel": ["NIGHTLY", "BETA", "RELEASE"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "EpsilonGreedyBanditStudy",
```
Comment (moritzhaller) [inline seed/seed.json:549]: Add `BraveAds.` as prefix so we append to confirmations

Comment (LorenzoMinto) [reply seed/seed.json]: 👍

**Review by moritzhaller — APPROVED** [review]
LGTM

**Comment by LorenzoMinto** [comment]
From `show-variations`
```
AdServingStudy:MaximumAdNotificationsPerDay=60
BraveAds.EpsilonGreedyBanditStudy:Treatment
BraveAds.FrequencyCappingStudy:ExcludeAdIfWithinTimeWindow=0h
BraveAds.ShowCustomAdNotificationOnMacStudy:CustomAdNotificationDefaultPosition
BraveAds.UserActivityStudy:Triggers=EMPTY/Threshold=0.0/IdleTimeThreshold=5
ConversionsParametersStudy:NewDefaultPattern
EphemeralStorageStudy:Enabled
PermissionLifetimeStudy:Enabled
```

**Pull request merged**